motion_computation/.clang-format

```diff
@@ -0,0 +1,46 @@
+---
+# ROS 2 C++ style guidelines
```

Comment:
Is there a reason we are using a different `.clang-format` here than in some of our other packages (i.e. `carma_cooperative_perception`, `roadway_objects`, etc.?

Reply:
This one is mostly same with `carma_cooperative_perception` except `SpacesBeforeTrailingComments: 2` and `ordering of the include files`. `SpacesBeforeTrailingComments: 2` was being complained in the cpplint in one of the runs:
![image](https://github.com/usdot-fhwa-stol/carma-platform/assets/20613282/0859530f-0fac-4fac-959b-43ffaf3b492e)

https://github.com/usdot-fhwa-stol/carma-platform/actions/runs/9486323826/job/26140411232

You can also see it is complaining about the orders of the include headers. I tried multiple times and think I achieved with the order here I got from autoware. but it was still complaining so I added extra file to disable that check just like `carma_cooperative_perception`